### PR TITLE
feat: modelo de datos

### LINKED ISSUE #2: Crear modelo de datos de acuerdo a todas las caracteristicas de una mascota y una fundación
**Requerimiento**

A continuación enumero algunos datos relevantes de cada mascota, estos se deben organizar y se debe crear un modelo en objeto JSON ya que será lo que utilizaremos para generar el API.

- Tipo
- Destacado 
- Genero
- Nombre 
-  Edad en meses 
-  Color 
- Desparasitado 
-  Libre Virales
-  Esterelizada
-  Cualidades 
- Necesidades especiales 
-  Historia
- Foto principal
- Galeria de fotos

También se debe tener en cuenta que esta mascota pueda estar asociada a una fundación

La fundación puede tener datos de:
- Nombre fundación
- Redes sociales
  - Instagram
  - Whatsapp
  - Sitio Web
  - Correo
- Objetivo / Misión
- Nombre contacto
- Historia
- Cuentas para adopción

Con el fin de garantizar una compabitiblidad con cualquier lenguaje, el modelo de datos debe ser creado en inglés.

**Resultado esperado**

- Un objeto JSON con la representación del modelo
- El objeto debe estar en inglés
- Se debe tener todas las propiedades descritas anteriormente
- Para presentar el modelo de datos se debe crear un objeto con extensión .json y debe tener datos de ejemplo para validar que el modelo sea correcto y cumpla las expectativas

### PR DESCRIPTION
**Pull request by aleocampodev**

### CODE REVIEW

File: model/foundation.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
+{
+	"foundations":[
+		{	
+			"id": 1,
+			"nameFoundation":"fundacion animal love",
```
Comment (codesandtags) [inline model/foundation.json:5]: Change value to `foundationName`

File: model/foundation.json [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+			"mision":"contribuir en ayudar a peluditos , para que encuentren un hogar y les brinden mucho amor,ya que son angeles terrenales. ",
```
Comment (codesandtags) [inline model/foundation.json]: Update with `mission`

File: model/foundation.json [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+			"history":"nuestra fundacion nace , gracias a nuestra veterinaria, quien inicia con esta bella obra.",
+			"contactname":"maria sofia gonzalez",
```
Comment (codesandtags) [inline model/foundation.json]: Please review the camelCase: `contactName`

File: model/foundation.json [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+			"accountsforadoption":[
```
Comment (codesandtags) [inline model/foundation.json]: Update this to: ´adoptionAccounts´

File: model/foundation.json [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+				{
+					"bancolombia": "b-032567489 "
+				},
+				{
+					"bancopopular": "974564"
+				}
+			],
+			"socialnetworks":[
```
Comment (codesandtags) [inline model/foundation.json]: Add camelCase: `socialNetworks`

File: model/foundation.json [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+			"pets":[
+				{
+					"id": 1,
+					"type": "gato",
+					"gender": "hembra" ,
+					"name": "brisa",
+					"age": "24 meses",
+					"color": "negro con blanco",
+					"history": "esta gatica fue encontrada en un lugar abandonado de la ciudad,donde personas 'humanas' ,contribuyeron para llegar a ella",
+					"mainphoto":"https://www.gatosexoticosweb.com/wp-content/uploads/2019/09/bicolor-blanco-y-negro.jpg",
```
Comment (codesandtags) [inline model/foundation.json]: Update with camelCase: `mainPhoto`

File: model/foundation.json [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+					"photogallery":[
```
Comment (codesandtags) [inline model/foundation.json]: Update to: `gallery`

File: model/foundation.json [hunk 8]
```diff
@@ -0,0 +1,77 @@
...
+						{
+							"url":"https://www.gatosexoticosweb.com/wp-content/uploads/2019/09/bicolor-blanco-y-negro.jpg"
+						},
+						{
+							"url":"https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRAK_H8rhKPnYj9I7ig6Y063oCJKb1SiSynEw19OuDeBDOCdq_y1rqQ0AxEf0A7HlVF_gU&usqp=CAU"
+						}
+					],
+					"dewormed":"true",
```
Comment (codesandtags) [inline model/foundation.json]: If this is a boolean value, you shouldn't use quotes: "true" -> true

File: model/foundation.json [hunk 9]
```diff
@@ -0,0 +1,77 @@
...
+					"sterilized": "false",
```
Comment (codesandtags) [inline model/foundation.json]: If this is a boolean value, you shouldn't use quotes: "true" -> true

File: model/foundation.json [hunk 10]
```diff
@@ -0,0 +1,77 @@
...
+					"freeviral": "negative",
```
Comment (codesandtags) [inline model/foundation.json]: If this is a boolean value, you shouldn't use quotes: "true" -> true

Comment (codesandtags) [inline model/foundation.json]: Update freeviral to `noVirals`

File: model/foundation.json [hunk 11]
```diff
@@ -0,0 +1,77 @@
...
+					"qualities": "cariñosa,independiente,inteligente,soñadora",
+					"specialneeds": "esta gatica necesita una dieta especial de alimento",
```
Comment (codesandtags) [inline model/foundation.json]: Add camelCase with `specialNeeds `

File: model/foundation.json [hunk 12]
```diff
@@ -0,0 +1,77 @@
...
+					"outstanding": "true"
```
Comment (codesandtags) [inline model/foundation.json]: Update outstanding to `highlighted` and also update the value to `true` or `false`

**Review by codesandtags — CHANGES_REQUESTED** [review]
Please review the comments I left

**Comment by aleocampodev** [comment]
feat: Se hacen los cambios pertinentes

**Pull request merged**